plot-doc/plot/scribblings/utils.scrbl

```diff
@@ -251,6 +258,28 @@ Integer brush styles repeat starting at @(racket 7).
                  (map ->brush-style '(4 5 6))]
 }
 
+@defproc[(color-map-names) (listof symbol?)]{
+
+Return the list of available color map names to be used by
+@racket[plot-pen-color-map] and @racket[plot-brush-color-map].
+
+}
+
+@defproc[(color-map-size (name symbol?)) integer?]{
+
+Return the number of colors in the color map @racket[name].  If @racket[name]
+is not a valid color map name, the function will signal an error.
+
+}
+
+@defproc[(register-color-map (name symbol?) (color-map (vectorof (list byte? byte? byte?)))) void]{
+
+Register a new color map @racket[name] with the colors being a vector of RGB
+triplets.  If a color map by that name already exists, it is replaced.
+
```

Comment:
Can you add @history annotations on these new functions?

Reply:
Will do that in a separate pull request